### ISSUE TITLE
Visualize cannot display big shared dimension in a map

### ISSUE THREAD

**Issue by CDiGallo** [visualize-admin/visualization-tool#1106]
Edit:
In the meantime I totally destroyed the cube in this post. The problem stays the same but please use the link from the first comment below.

**Describe the bug**
Shared dimension municipalities cannot get loaded as a map on visualize. Even on 1 Municapilites i waited longer nthan 270 min.
The other charts, such as bar chart work in <1 second.

**To Reproduce**

1. for cantons it works: https://int.[visualize.admin.ch](https://int.visualize.admin.ch/v/52lYVlmgczBm?dataSource=Int)/v/52lYVlmgczBm?dataSource=Int
2. for municpalities not: https://int.visualize.admin.ch/en/create/2BeD8NYUJLMG?dataSource=Int
3.  Click on bar chart -> works
4. click on maps -> (probably) infinite loading

**Expected behavior**
see the canton visualization. I want to have the municipalities in the color based on a measure-dimension.

**Screenshots or video**
If applicable, add screenshots or a short video to help put your problem into a context.

**Environment (please complete the following information):**

- Visualize environment and version: INT
- Browser and version EDGE

**Additional context**

**Comment by CDiGallo** [comment]
I made some further investigations.
I tested in an extra [cube](https://test.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!municpaltiestest4entries-l0yd0gwej07/publication) how visualize behaves with less municipalities (so confirming, that it is actually a perfomance issue).

with less than a hundred cases, it is very fast.
With 1.5k municipalties it has around 5 mins or a bit less, but it still works. -> https://test.visualize.admin.ch/de/v/yof4TUv0bmCU

Just for added information
